internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -17,6 +17,21 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
+// aux methods to convert nil <-> empty strting
+func PtrOrNil(s string) *string {
```

Comment:
Why not use [sql.NullString](https://pkg.go.dev/database/sql#NullString) ?

Reply:
ptr to string is simpler. sql.NullString will do the same in background, but require more changes in the code.